using-kubecost/navigating-the-kubecost-ui/cloud-costs-explorer/cloud-cost-metrics.md

```diff
@@ -110,15 +110,17 @@ The Cost column plus the sum of all credit amounts.
 
 **Amortized Net Cost**
 
-Kubecost uses Net Cost.
+Amortized Net Cost is Cost with all credits and amortized CUD payments
+`amortizedNetCost := cost + creditAmount + cudCreditAmount + flexibleCUDCreditAmount + flexibleCUDNetPayedAmount`
```

Comment:
Since the cloud cost pipeline code is open source, we could add a reference pointing directly to the source code for users that are interested in the in-depth implementation details. I think the doc can be refactored to add the notes on how each cost type is computed, but I don't think we need to show the actual equations. That's just a personal opinion though, deferring to @bstuder99 here!

Reply:
I think both is always good if we can provide it, I like the addition of the source code

Reply:
I think having the equations is good, personally it makes it easier to understand the math.